app/src/main/res/layout/checked_list_item.xml

```diff
@@ -8,4 +8,5 @@
     android:paddingStart="25dp"
     android:paddingEnd="10dp"
     android:textColor="@color/daynight_textColor"
-    android:textSize="16sp" />
+    android:textSize="16sp"
+    android:checkMarkTint="@color/daynight_textColor"/>
```

Comment:
to follow the convention leave space between closing bracket and the text
`android:checkMarkTint="@color/daynight_textColor" />`